pkg/perf/cache.go

```diff
@@ -0,0 +1,43 @@
+package perf
+
+import (
+	"context"
+	"encoding/json"
+
+	"github.com/pkg/errors"
+)
+
+// TaskResult the result test schema
+type TaskResult struct {
+	TaskName     string      `json:"task_name"`
+	RunTimestamp string      `json:"run_timestamp"`
```

Comment:
Please change this to an actual unix timpestamp uint64. Storing time as a string is the most inefficient way to store time.